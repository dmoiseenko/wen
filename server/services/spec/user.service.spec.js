const userService = require('../user.service');
const bootstrap = require('../../db/bootstrap/bootstrap.js');


describe('getById', () => {
  beforeEach(() => bootstrap());

  it('should get user by id', async () => {
    const actual = await userService.getById(1);

    delete actual.passwordHash;

    expect(actual).toMatchSnapshot();
  });
});
