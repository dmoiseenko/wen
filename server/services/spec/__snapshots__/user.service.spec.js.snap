// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getById should get user by id 1`] = `
Object {
  "email": "js@mail.com",
  "firstName": "John",
  "id": 1,
  "lastName": "Smith",
}
`;
